koku/masu/test/database/test_ocp_markup_concurrency.py

```
@@ -0,0 +1,85 @@
+        results = [None, None]
+
+        def run_markup(index, markup_value):
+            try:
+                barrier.wait()
+                with tenant_context(self.tenant):
+                    accessor = OCPReportDBAccessor(schema=self.schema)
+                    accessor.populate_markup_cost(
+                        Decimal(markup_value), self.start_date, self.end_date, self.cluster_id
+                    )
+                results[index] = "ok"
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)